perllib/FixMyStreet/Cobrand/Peterborough.pm

```diff
@@ -1267,6 +1365,27 @@ sub open311_contact_meta_override {
     }
 }
 
+sub waste_munge_bulky_cancellation_data {
+    my ( $self, $data ) = @_;
+
+    my $c = $self->{c};
+
+    $data->{title}    = 'Bulky goods cancellation';
+    $data->{category} = 'Bulky cancel';
+    $data->{detail} .= " | Original report ID: "
+        . $c->stash->{property}{pending_bulky_collection}->id;
+
+    $data->{extra_COMMENTS} = 'Cancellation at user request';
```

Comment:
Don't think we need to set this in $data, can just be set in the param next line.